src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Please rewrite all the styles using the `styled-components` https://styled-components.com/docs